app/controllers/policies_controller.rb

```diff
@@ -34,44 +50,8 @@ def policies_params
      address_attributes: [:city,:state,:pincode,:street_1,:street_2],
      life_insurance_attributes: [:policy_term,:education_qualification,:annual_income,:term_rider,:critical_illness,:with_aaccident_cover])
   end
-end
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
 
+  def find_by_policy_id
```

Comment:
@GauravMakwana-BTC : It should be set_policy as per rails standard, update this other places as well